Ports the RBMK from NSV

XeonMations opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

Adds in the RBMK nuclear reactor from NSV.

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->
With the recent atmos bugs going on and general engineering incompetence, why not add another engine for engineering to fuck up and blow a hole in the engineering department with.

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62395746/2b621880-aee0-4b34-ae57-7a201f8c6804)
![image-1](https://github.com/BeeStation/BeeStation-Hornet/assets/62395746/a4a38da7-bc0f-43a4-8442-3d71fb138f5a)
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62395746/ab5b8afd-fb2f-434e-8e86-e7cf397694a9)
![image-1](https://github.com/BeeStation/BeeStation-Hornet/assets/62395746/44e79701-2b5f-4821-842d-fbe18cfabb30)



</details>

## Changelog
:cl: Xeonmations, Aramix
add: Added the RBMK reactor, ported from NSV.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

XeonMations committed:
RBMK has arrived

my sanity is gone

XeonMations committed:
Fixed TGUI computers

Should fix the checks

XeonMations committed:
Todo list updated + design change

XeonMations committed:
Fixes and changes

Removed control rods due to them being not involved with RBMK
Added fuel rods and sealant to cargo order list
Increased blowout condition spreading slag

XeonMations committed:
Final

Fixed reactor audio warning, fixed computers not connecting to reactor on creation, removed snowflake ambience code, made light flickers more obvious during overload

XeonMations committed:
Changed radstation to be a TM map

also fixed reactor computers

Penwin0 commented on code/game/objects/effects/decals/cleanable/misc.dm:
```diff
@@ -216,3 +216,4 @@
 	icon = 'icons/effects/blood.dmi'
 	icon_state = "xfloor1"
 	random_icon_states = list("xfloor1", "xfloor2", "xfloor3", "xfloor4", "xfloor5", "xfloor6", "xfloor7")
+
```
```suggestion
```

XeonMations committed:
Update code/game/objects/effects/decals/cleanable/misc.dm

Co-authored-by: Penwin <53474257+Penwin0@users.noreply.github.com>

XeonMations committed:
Fix run_linters

XeonMations committed:
Merge branch 'RBMK' of https://github.com/XeonMations/BeeStation-Hornet into RBMK

JixS4v commented on code/modules/power/rbmk/rbmk.dm:
```diff
@@ -0,0 +1,958 @@
+		if(power >= 20)
+			to_chat(user, "<span class='notice'>You cannot repair [src] while it is running at above 20% power.</span>")
+			return FALSE
+		if(vessel_integrity >= 350)
```
```suggestion
		if(vessel_integrity >= 0.875*initial(vessel_integrity))
```
Seems a bit weird to base some stuff on the initial vessel integrity and other stuff on absolute values, better to standardize.

JixS4v commented on code/modules/power/rbmk/rbmk.dm:
```diff
@@ -0,0 +1,958 @@
+			to_chat(user, "<span class='notice'>[src]'s reactor vessel is cracked and worn, you need to repair the cracks with a welder before you can repair the seals.</span>")
+			return FALSE
+		if(do_after(user, 5 SECONDS, target=src))
+			if(vessel_integrity >= 350)	//They might've stacked doafters
```
```suggestion
			if(vessel_integrity >= 0.875*initial(vessel_integrity))	//They might've stacked doafters
```

XeonMations committed:
Update code/modules/power/rbmk/rbmk.dm

Co-authored-by: JixS4v <61665800+JixS4v@users.noreply.github.com>

XeonMations committed:
Update code/modules/power/rbmk/rbmk.dm

Co-authored-by: JixS4v <61665800+JixS4v@users.noreply.github.com>

XeonMations committed:
fixes weird tests

they were working before, idk why they stopped working, but this seems to be the issue? maybe it'll work now

XeonMations committed:
hopefully this reverts the TM map

XeonMations committed:
Update RadStation.dmm

XeonMations committed:
okay maybe THIS fixed it

XeonMations committed:
thanks floria for the help

XeonMations committed:
NOW its fixed

XeonMations committed:
pigeon made me do this
